lib/pages/movie_menu.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import 'package:movie_card/shared/movie_class.dart';
+import 'package:movie_card/widgets/menu/header.dart';
+import 'package:movie_card/shared/ui_consts.dart';
+import 'package:movie_card/widgets/menu/last_seen.dart';
+import 'package:movie_card/widgets/menu/menu_custom_navbar.dart';
+import 'package:movie_card/widgets/menu/movies.dart';
+
+class MovieMenu extends StatefulWidget {
+  const MovieMenu({super.key});
+
+  @override
+  State<MovieMenu> createState() => _MovieMenuState();
+}
+
+class _MovieMenuState extends State<MovieMenu> {
+  final List<Movie> movies = List.generate(
```

Comment:
Why you are generating a list of movies, the idea to this exercise is to have only one movie